src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User | null;
+};
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
```

Comment:
It's recommended to add a fallback for when the user is null or undefined. You can use the `??` operator to provide a default value 🚀